src/Gaffer/Animation.cpp

```diff
@@ -156,6 +156,40 @@ double slopeFromPosition( const Imath::V2d& position, const Gaffer::Animation::D
 	}
 }
 
+bool tieSlopeActive( const Gaffer::Animation::TieMode tieMode )
```

Comment:
Quick comment while I'm looking through these fixups:  this is fine, but I think I would find this clearer than the switch as a simple expression:
`return tieMode == Gaffer::Animation::TieMode::Slope || tieMode == Gaffer::Animation::TieMode::Scale;`

Reply:
done